frontend/app/routes/protected/person-case/review.tsx

```diff
@@ -66,8 +67,11 @@ export async function action({ context, params, request }: Route.ActionArgs) {
     }
 
     case 'next': {
+      const authToken = context.session.authState.idToken;
+      invariant(authToken, 'Expected authToken to be defined');
```

Comment:
This should probably throw an AppError with a `opts.httpStatusCode = HttpStatusCodes.FORBIDDEN` (or even UNAUTHORIZED)

Reply:
Will resolve this. We can look at improving it later.